content/en/docs/reference/setup-tools/kubeadm/kubeadm-join.md

```diff
@@ -41,6 +34,40 @@ This action consists of the following steps:
 1. Finally, kubeadm configures the local kubelet to connect to the API
    server with the definitive identity assigned to the node.
 
+For control-plane nodes additional steps are performed:
+
+1. Downloading certificates shared among control-plane nodes from the cluster
+  (if explicitly requested by the user).
+
+1. Generating control-plane component manifests, certificates and kubeconfig.
+
+1. Adding new local etcd member.
+
+1. Adding this node to the ClusterStatus of the kubeadm cluster.
+
+### Using join phases with kubeadm {#join-phases}
+
+Kubeadm allows you join a node to the cluster in phases. The `kubeadm join phase` command was added in v1.14.0.
+
+To view the ordered list of phases and sub-phases you can call `kubeadm join --help`. The list will be located
+at the top of the help screen and each phase will have a description next to it.
+Note that by calling `kubeadm join` all of the phases and sub-phases will be executed in this exact order.
+
+Some phases have unique flags, so if you want to have a look at the list of available options add `--help`, for example:
+
+```bash
```

Comment:
nit:  would have been nice if you kept consistency with the other kubeadm pages where you used `shell`. Now i am aware this is just cosmetics but even so ..

Reply:
updated.